testing/cuda/memory.cu

```diff
@@ -58,8 +58,10 @@ void TestGetTemporaryBufferDeviceSeq()
   thrust::device_vector<ptr_and_sz_type> d_result(1);
   
   get_temporary_buffer_kernel<<<1,1>>>(n, d_result.begin());
-  cudaError_t const err = cudaDeviceSynchronize();
-  ASSERT_EQUAL(cudaSuccess, err);
+  {
+    cudaError_t const err = cudaDeviceSynchronize();
+    ASSERT_EQUAL(cudaSuccess, err);
+  }
```

Comment:
Why put this in a scope?

Reply:
Avoid conflicts with later declarations of `err`.